pandas/tests/frame/test_apply.py

```diff
@@ -481,6 +481,99 @@ def test_apply_non_numpy_dtype(self):
         result = df.apply(lambda x: x)
         assert_frame_equal(result, df)
 
+    def test_gh_19359_with_and_without_tz(self):
+        # GH #19359
+        def transform_time(x):
+            from dateutil.parser import parse
+            return Series({'time': parse("22:05 UTC+1"),
+                           'title': parse("23:59")})
+
+        applied = DataFrame(["stub"]).apply(transform_time)
```

Comment:
``result = ``

construct the expected directly

Reply:
done